astroquery/vizier/core.py

```diff
@@ -370,22 +373,35 @@ def query_region_async(self, coordinates, radius=None, inner_radius=None,
 
         # Process coordinates
         if isinstance(coordinates, (commons.CoordClasses,) + six.string_types):
-            c = commons.parse_coordinates(coordinates).transform_to('fk5')
+            c = commons.parse_coordinates(coordinates).transform_to(frame)
```

Comment:
unrelated to this PR, but it's a good time to rename `c` to something more verbose :)